CRMImporter/ActionHandlers/EmptyHandler.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xrm.Sdk;
+
+namespace CRMImporter.ActionHandlers
+{
+    public class EmptyHandler : IActionHandler

```

Comment:
What is the purpose of EmptyHandler?  It doesn't seem to be used anywhere outside your tests.

Reply:
If people want to disable either the create or update functionality. For example if you don't want any new entities to be created you could just replace the create handler with the empty handler.